src/data/list/rotate.lean

```diff
@@ -347,6 +381,12 @@ h.perm.mem_iff
 @[simp] lemma is_rotated_nil_iff' : [] ~r l ↔ l = [] :=
 by rw [is_rotated_comm, is_rotated_nil_iff]
 
+@[simp] lemma is_rotated_singleton_iff {x : α} : l ~r [x] ↔ l = [x] :=
+⟨λ ⟨n, hn⟩, by simpa using hn, λ h, h ▸ by refl⟩
+
+@[simp] lemma is_rotated_singleton_iff' {x : α} : [x] ~r l ↔ l = [x] :=
```

Comment:
Again, I think making the LHS and RHS look more similar is probably sensible here:
```suggestion
@[simp] lemma is_rotated_singleton_iff' {x : α} : [x] ~r l ↔ [x] = l :=
```